src/lib/util/style-object-to-string.ts

```diff
@@ -0,0 +1,5 @@
+export function styleObjectToString(styleObject: Partial<ElementCSSInlineStyle['style']>) {
```

Comment:
If we think most utility functions will be small, let's condense down to `/lib/util/util.ts` (or similar) to avoid premature abstraction.